app/javascript/courses/review/components/CoursesReview__ChecklistEditor.res

```diff
@@ -226,19 +307,41 @@ let make = (~reviewChecklist, ~updateReviewChecklistCB, ~closeEditModeCB, ~targe
                                   )}
                               />
                               <div
-                                className="flex w-10 h-10 absolute top-0 right-0 mr-1 items-center justify-center">
-                                <button
-                                  title={t("checklist_item_title.remove_button_title")}
-                                  className="flex items-center justify-center bg-gray-100 w-7 h-7 mt-px text-sm text-gray-700 hover:text-red-600 hover:bg-red-100 rounded-full ml-2 border border-transparent text-center"
-                                  onClick={_ =>
-                                    removeChecklistResult(
-                                      itemIndex,
-                                      resultIndex,
-                                      reviewChecklistItem,
-                                      setState,
-                                    )}>
-                                  <Icon className="if i-times-regular" />
-                                </button>
+                                className="flex h-10 absolute top-0 right-0 mr-1 items-center justify-center">
+                                {controlIcon(
+                                  ~icon="fa-arrow-up",
+                                  ~title={t("checklist_item_title.move_up_button_title")},
+                                  ~handler=moveChecklistResultUp(
+                                    itemIndex,
+                                    resultIndex,
+                                    reviewChecklistItem,
+                                    setState,
+                                  ) |> OptionUtils.map((cb, _) => cb()),
+                                )}
+                                {controlIcon(
+                                  ~icon="fa-arrow-down",
+                                  ~title={t("checklist_item_title.move_down_button_title")},
+                                  ~handler=moveChecklistResultDown(
+                                    itemIndex,
+                                    resultIndex,
+                                    reviewChecklistItem,
+                                    reviewChecklistItem.result,
```

Comment:
We will always use functions defined in the type file for accessing properties from the record. 
```suggestion
                                    ReviewChecklistItem.result(reviewChecklistItem),
```